src/client/components/MyProfile/YourUploadedCVs/Star.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './YourUploadedCVs.css';
+
+export function Star({ averageRatingInPercent }) {
+  return (
+    <div className="ratings">
+      <div className="empty-stars" />
+      <div
+        className="full-stars"
+        style={{ width: `${averageRatingInPercent}` }}
```

Comment:
Thy is the width equal to averageRatingInPercent? I dont really understand that

Reply:
I just define this probs to use it in the RatingStars component. when I need to bring filled stars I juist define it 100% and for the rest sar that may be is less than 1 I define it upon my desire. for example if the evarge is 3.75. I bring this probs 3 times with 100% width and one time with 75% width.